src/components/custom/signup.tsx

```diff
@@ -92,15 +92,34 @@ const Signup = (props: SignupProps) => {
         return error_message
     }
 
+    // this method is used to prevent DOM XSS invulnerability by removing HTML DOM elements in the email input
+    function escapeHtml(value: string): string {
+        return value.replaceAll(/[&<>"'“”‘’]/g, (c) => {
+            switch (c) {
+                case '&':
+                case '<':
+                case '>':
+                case '"':
+                case "'":
+                case '“':
+                case '”':
+                case '‘':
+                case '’':
```

Comment:
Added checks for special quotation characters available in IOS keyboard `‘ ’ “ ”`